galleon-pack/src/main/resources/layers/standalone/observability/layer-spec.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" ?>
 <layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="observability">
     <dependencies>
-        <layer name="microprofile-config"/>
-        <layer name="microprofile-metrics"/>
+        <layer name="microprofile-config" optional="true"/>
```

Comment:
I'm reluctant to make config optional. The odds are very high that someone trying to exclude it will generate a broken config.  OpenTracing doesn't require it, but if all you want is OT then just specify that.